docker/quickstart/Dockerfile

```diff
@@ -66,4 +67,4 @@ ENV REINDEX_DATASETS=0
 
 ENV ES_JAVA_OPTS=-'Xms512m -Xmx512m'
```

Comment:
This memory restriction could be related with ElasticSearch problems in HF. We should take a look after confirming the error on HF.